docs/installation.md

```diff
@@ -261,3 +261,12 @@ visible to the classloader which currently locates the javac Main class. Then
 replace the call of `javac.main.Main.main()` with the Error Prone compiler:
 
 `return new ErrorProneCompiler.Builder().build().compile(args) == 0`
+
+## Using ErrorProne in concert with other Annotation Processors
+
+All of the above instructions use the `javac` option `-processorpath` which has
+side-effect of causing `javac` to no longer scan the compile classpath for
+annotation processors. If you are using other annotation processors in addition
+to ErrorProne, such as Lombok, then you will need to add their JARs to your
+`-processorpath` argument. The mechanics of this will vary according to the build
+tool you are using.
```

Comment:
…too bad this does not list them :wink: 

How about adding a comment to the snippets above saying something like "insert your other annotation processors here"?
(though that this applies mainly to Maven; it gained only recently the ability to use the processor path, and almost nobody updated their build, and most importantly their knowledge)

Reply:
Good point
I did similar to that in the adjacent PR #1187
I will add the same change to Maven above.
I'll have a go at the others, but Bazel and Gradle look like closed black-boxes to me that don't admit such a change at present

Reply:
> Maven gained only recently the ability to use the processor path, and almost nobody updated their build, and most importantly their knowledge

Why use "-processorpath" instead of adding these things to the main classpath in "provided" scope? It seems to me that the latter is more robust and much easier when multiple annotation processors are in use.

Reply:
> Why use "-processorpath" instead of adding these things to the main classpath in "provided" scope? It seems to me that the latter is more robust and much easier when multiple annotation processors (or custom ErrorProne rules) are in use.

ErrorProne is a plugin (`-Xplugin`), and those are **only** loaded from the processor path, not the compilation classpath.

Wrt other usages of annotation processors, without ErrorProne in loop, having too separate classpaths means you can have separate dependencies, or put differently, that annotation processors won't impose their own dependencies on you. For example, if you use Guava and target Java 7, you won't use a version past 20.0 (or so, from memory); or if you target Android you'll use a special version targeting Android; you don't want your annotation processors (such as Dagger or AutoValue) to impose a different or more recent version of Guava. Putting them in the processor path helps segregate those dependency trees.

Reply:
Thanks, that makes sense